README.md

```diff
@@ -108,20 +108,24 @@ When running `modelCheckingTest(),` Lincheck not only detects a bug but also pro
 
 ```text
 = Invalid execution results =
-Init part:
-[addLast(4): void]
-Parallel part:
-| pollFirst(): 4 | addFirst(-4): void       |
-|                | peekLast():   4    [-,1] |
+| -------------------------------------- |
+|     Thread 1     |      Thread 2       |
+| -------------------------------------- |
+| addLast(4): void |                     |
+| -------------------------------------- |
```

Comment:
Let's explain in the text output what the horizontal line means.

Reply:
What message do you want to put here? What should be the formatting? Can you please give an example.

Reply:
Let's add a hint (like the one about the clocks) indicating that all operations above the horizontal line occur before those below it.